CP-CPS.md

```diff
@@ -520,7 +524,9 @@ ISRG allows for OCSP stapling.
 
 ### 4.9.12 Special requirements re key compromise
 
-Key compromise must be demonstrated via the Certificate Revocation method of the ACME Protocol defined in RFC 8555, Section 7.6 by signing the request using the private key corresponding to the public key in the certificate (not the private key corresponding to the account which originally requested the certificate).
+In order for a successful revocation request to result in the blocking of a key for future issuance and revocation of all currently valid certificates with that key, proof of possession of the private key for the certificate must be demonstrated.
+
+Proof of possession of a private key corresponding to the public key of a certificate must be demonstrated via the Certificate Revocation method of the ACME Protocol defined in RFC 8555, Section 7.6 by signing the request using the private key corresponding to the public key in the certificate (not the private key corresponding to the account which originally requested the certificate). 
```

Comment:
A version of this which should convey all the same information but is more in keeping with the brevity of the original.

```suggestion
To block a key for future issuance and revoke all valid certificates associated with that key, the requester must demonstrate possession of the certificate’s private key. This proof must be provided via the Certificate Revocation method in the ACME Protocol, as defined in RFC 8555, Section 7.6. The revocation request must be signed using the private key that corresponds to the certificate's public key. This key must be distinct from the private key associated with the account that originally requested the certificate.
```